readthedocs/domains/templates/domains/notifications/pending_domain_configuration_site.html

```diff
@@ -0,0 +1,7 @@
+{% url "projects_domains_edit" domain.project.slug domain.pk as domain_url %}
+{% if domain.validation_process_expired %}
+  The validation period of your custom domain <a href="{{ domain_url }}">{{ domain.domain }}</a> has ended.
+  Save the domain to restart the process.
```

Comment:
Something more direct here will work better:

```suggestion
  Go to <a>domain configuration</a> and click on "Save" to restart the process.
```